designer/client/src/components/toolbarSettings/actions.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
We already have some file with constant values `designer/client/src/stylesheets/variables.ts`. Maybe, it will be a better place for it?

Reply:
Fixed